src/ui/scss/component/_comments.scss

```diff
@@ -0,0 +1,67 @@
+.comment {
+  padding-bottom: var(--spacing-vertical-large);
+
+  &.reply {
```

Comment:
this should be the class `comment--reply` and be a first-class rule (i.e. not nested) if we're following BEM